lxd/network/driver_common.go

```diff
@@ -578,15 +578,24 @@ func (n *common) handleDependencyChange(netName string, netConfig map[string]str
 func (n *common) bgpValidationRules(config map[string]string) (map[string]func(value string) error, error) {
 	rules := map[string]func(value string) error{}
 	for k := range config {
-		// BGP keys have the peer name in their name, extract the suffix.
+		// only BGP keys are handled here.
 		if !strings.HasPrefix(k, "bgp.") {
```

Comment:
Thanks @tomponline for the clarification. 

@mandrav then I would suggest to revert the changes (you can leave the `%q` for the error message) and extend this line to just check for `bgp.peers.` which will exclude the two nexthop settings.

```suggestion
		if !strings.HasPrefix(k, "bgp.peers.") {
```